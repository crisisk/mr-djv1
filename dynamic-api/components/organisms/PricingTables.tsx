'use client';

import React from 'react';
import Button from '../ui/Button';
import { trackEvent } from '../../lib/analytics'; // Assuming this utility exists

// --- Interfaces ---

interface Upsell {
  id: string;
  label: string;
  price: string;
}

interface Package {
  name: string;
  subtitle: string;
  price: string;
  duration: string;
  category: string[];
  features: string[];
  upsells: Upsell[];
  isFeatured: boolean;
  buttonText: string;
  badge?: string;
}

interface PricingCardProps {
  pkg: Package;
  segmentId?: string;
  onSelect?: (pkg: Package) => void;
}

// --- Data ---

const packages: Package[] = [
  {
    name: 'Brons',
    subtitle: 'Entry-level pakket',
    price: '€495',
    duration: '4 uur showtijd',
    category: ['Private', 'Bedrijfsfeest'],
    features: [
      '4 uur DJ-set',
      'Basis licht- en geluidsset',
      'Persoonlijk intakegesprek',
      'Muziekvoorkeuren formulier',
    ],
    upsells: [
      { id: 'rookmachine', label: 'Rookmachine & sfeerlicht', price: '€75' },
      { id: 'ceremonie', label: 'Ceremonie audio set', price: '€195' },
    ],
    isFeatured: false,
    buttonText: 'Meer Info',
  },
  {
    name: 'Zilver',
    subtitle: 'Meest gekozen',
    price: '€795',
    duration: '5 uur showtijd',
    category: ['Bruiloft'],
    features: [
      '6 uur DJ-set',
      'Uitgebreide lichtshow',
      'DJ + Saxofonist optie',
      '100% dansgarantie',
      'Onbeperkt aantal gasten',
    ],
    upsells: [
      { id: 'sparkular', label: 'Sparkular show', price: '€245' },
      { id: 'sax', label: 'Live saxofonist', price: '€325' },
      { id: 'photobooth', label: '360° photobooth', price: '€425' },
    ],
    isFeatured: true,
    buttonText: 'Boek Nu',
  },
  {
    name: 'Goud',
    subtitle: 'Premium All-Inclusive',
    price: '€1.295',
    duration: '6 uur showtijd + productie',
    category: ['Bruiloft', 'Bedrijfsfeest'],
    features: [
      '8 uur DJ-set',
      'Luxe licht- en geluidsset',
      'DJ + Saxofonist (inbegrepen)',
      'Ceremonie & receptie muziek',
      'Professionele apparatuur',
    ],
    upsells: [
      { id: 'branding', label: 'Custom booth branding', price: '€395' },
      { id: 'host', label: 'Host & vocalist', price: '€295' },
      { id: 'afterfilm', label: 'Aftermovie & content', price: '€495' },
    ],
    isFeatured: false,
    buttonText: 'Vraag Offerte Aan',
  },
];

// --- Components ---

const PricingCard: React.FC<PricingCardProps> = ({ pkg, segmentId, onSelect }) => {
  const { name, subtitle, price, duration, category, features, upsells, isFeatured, buttonText, badge } = pkg;

  const cardClasses = isFeatured
    ? 'bg-primary text-white shadow-2xl transform scale-105 ring-4 ring-secondary/50'
    : 'bg-white text-neutral-dark shadow-lg ring-1 ring-neutral-gray-100';

  const headerClasses = isFeatured
    ? 'text-secondary border-b border-secondary/50'
    : 'text-primary border-b border-neutral-gray-100';

  const buttonVariant = isFeatured ? 'secondary' : 'primary';

  const handleClick = () => {
    if (typeof trackEvent === 'function') {
      trackEvent('package_cta_click', {
        package_name: name,
        placement: 'design_system_pricing_tables',
        personalization_variant: segmentId || 'default',
      });
    }
    onSelect?.(pkg);
  };

  const handleHover = () => {
    if (typeof trackEvent === 'function') {
      trackEvent('package_view', {
        package_name: name,
        placement: 'design_system_pricing_tables',
        personalization_variant: segmentId || 'default',
      });
    }
  };

  return (
    <div
      className={`relative flex flex-col p-spacing-xl rounded-lg transition duration-300 h-full ${cardClasses}`}
      onMouseEnter={handleHover}
      aria-label={`Pakket: ${name}`}
    >
      {isFeatured && (
        <div className="absolute top-0 right-0 -mt-4 -mr-4 px-spacing-md py-spacing-xs bg-secondary text-white font-bold rounded-full shadow-md uppercase text-sm tracking-wider">
          Meest Populair
        </div>
      )}

      <header className={`pb-spacing-lg mb-spacing-lg ${headerClasses}`}>
        <h3 className="text-2xl font-bold mb-spacing-xs">{name}</h3>
        <p className="text-sm opacity-80">{subtitle}</p>
        <p className="text-5xl font-extrabold mt-spacing-md">{price}</p>
        <p className="text-sm opacity-70 mt-spacing-xs">{duration}</p>
      </header>

      <div className="flex-grow">
        <ul className="space-y-spacing-sm mb-spacing-lg">
          {features.map((feature, index) => (
            <li key={index} className="flex items-start text-sm">
              <svg
                className={`w-5 h-5 mr-spacing-sm flex-shrink-0 ${isFeatured ? 'text-secondary' : 'text-primary'}`}
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
                aria-hidden="true"
              >
                <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M5 13l4 4L19 7" />
              </svg>
              {feature}
            </li>
          ))}
        </ul>

        <div className="mt-spacing-lg pt-spacing-lg border-t border-dashed border-current/30">
          <h4 className="font-semibold mb-spacing-sm text-sm uppercase opacity-80">Optionele Upgrades</h4>
          <ul className="space-y-spacing-xs text-sm opacity-90">
            {upsells.map((upsell) => (
              <li key={upsell.id} className="flex justify-between">
                <span>{upsell.label}</span>
                <span className="font-medium">{upsell.price}</span>
              </li>
            ))}
          </ul>
        </div>
      </div>

      <div className="mt-spacing-xl pt-spacing-lg">
        <Button
          variant={buttonVariant}
          size="lg"
          className="w-full"
          onClick={handleClick}
          aria-label={`Selecteer pakket ${name}`}
        >
          {buttonText}
        </Button>
      </div>
    </div>
  );
};

const PricingTables: React.FC = () => {
  const segmentId = 'pricing_page_v1'; // Example segment ID

  return (
    <section className="py-spacing-xl bg-neutral-gray-50" aria-labelledby="pricing-heading">
      <div className="max-w-7xl mx-auto px-spacing-lg">
        <h2 id="pricing-heading" className="text-4xl font-extrabold text-center text-neutral-dark mb-spacing-xl">
          Onze Pakketten
        </h2>
        <p className="text-center text-lg text-neutral-dark/80 mb-spacing-xl max-w-3xl mx-auto">
          Kies het pakket dat perfect aansluit bij uw evenement. Elk pakket is volledig aanpasbaar met onze optionele upgrades.
        </p>

        <div className="grid grid-cols-1 md:grid-cols-3 gap-spacing-lg items-stretch">
          {packages.map((pkg) => (
            <PricingCard
              key={pkg.name}
              pkg={pkg}
              segmentId={segmentId}
              onSelect={(selectedPkg) => console.log('Package selected:', selectedPkg.name)}
            />
          ))}
        </div>
      </div>
    </section>
  );
};

export default PricingTables;