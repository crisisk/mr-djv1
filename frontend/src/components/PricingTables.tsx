"use client";

import { Button } from './Button';

export interface PricingFeatureSet {
  name: string;
  subtitle: string;
  price: string;
  features: string[];
  isFeatured?: boolean;
  buttonText: string;
}

export interface PricingTablesProps {
  packages?: PricingFeatureSet[];
}

const euroFormatter = new Intl.NumberFormat('nl-NL', {
  style: 'currency',
  currency: 'EUR'
});

const defaultPackages: PricingFeatureSet[] = [
  {
    name: 'Brons',
    subtitle: 'Entry-level pakket',
    price: euroFormatter.format(495),
    features: ['4 uur DJ-set', 'Basis licht- en geluidsset', 'Persoonlijk intakegesprek', 'Muziekvoorkeuren formulier'],
    buttonText: 'Meer Info',
  },
  {
    name: 'Zilver',
    subtitle: 'Meest gekozen',
    price: euroFormatter.format(795),
    features: [
      '6 uur DJ-set',
      'Uitgebreide lichtshow',
      'DJ + Saxofonist optie',
      '100% dansgarantie',
      'Onbeperkt aantal gasten',
    ],
    isFeatured: true,
    buttonText: 'Boek Nu',
  },
  {
    name: 'Goud',
    subtitle: 'Premium All-Inclusive',
    price: euroFormatter.format(1295),
    features: [
      '8 uur DJ-set',
      'Luxe licht- en geluidsset',
      'DJ + Saxofonist (inbegrepen)',
      'Ceremonie & receptie muziek',
      'Professionele apparatuur',
    ],
    buttonText: 'Vraag Offerte Aan',
  },
];

function PricingCard({ pkg }: { pkg: PricingFeatureSet }) {
  const { name, subtitle, price, features, isFeatured, buttonText } = pkg;

  const cardClasses = isFeatured
    ? 'bg-neutral-dark text-neutral-light shadow-2xl transform scale-105'
    : 'bg-neutral-light text-neutral-dark shadow-lg';

  const headerClasses = isFeatured
    ? 'text-secondary border-b border-secondary/50'
    : 'text-primary border-b border-neutral-gray-100';

  const buttonVariant = isFeatured ? 'secondary' : 'primary';

  return (
    <div className={`relative flex flex-col p-spacing-xl rounded-lg transition duration-300 ${cardClasses}`}>
      {isFeatured ? (
        <div className="absolute top-0 right-0 bg-secondary text-neutral-dark body-sm text-strong px-spacing-md py-spacing-xs rounded-tr-lg rounded-bl-lg">
          Populair
        </div>
      ) : null}
      <div className={`pb-spacing-md mb-spacing-md ${headerClasses}`}>
        <h3 className="heading-3">{name}</h3>
        <p className="body-sm opacity-80">{subtitle}</p>
      </div>
      <div className="flex items-baseline mb-spacing-lg">
        <span className="heading-1">{price}</span>
        <span className="body-md ml-spacing-xs">/ event</span>
      </div>
      <ul className="flex-grow space-y-spacing-sm mb-spacing-xl">
        {features.map((feature) => (
          <li key={feature} className="flex items-start body-md">
            <svg
              className={`w-5 h-5 mr-spacing-sm ${isFeatured ? 'text-secondary' : 'text-primary'}`}
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M5 13l4 4L19 7" />
            </svg>
            {feature}
          </li>
        ))}
      </ul>
      <Button variant={buttonVariant} size="lg" className="w-full">
        {buttonText}
      </Button>
    </div>
  );
}

export function PricingTables({ packages = defaultPackages }: PricingTablesProps) {
  return (
    <section className="py-spacing-3xl bg-neutral-gray-100">
      <div className="container mx-auto px-spacing-md">
        <h2 className="heading-2 text-center text-neutral-dark mb-spacing-2xl">Onze Pakketten</h2>
        <div className="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl items-center">
          {packages.map((pkg) => (
            <PricingCard key={pkg.name} pkg={pkg} />
          ))}
        </div>
      </div>
    </section>
  );
}

export default PricingTables;
export { defaultPackages as defaultPricingPackages };
