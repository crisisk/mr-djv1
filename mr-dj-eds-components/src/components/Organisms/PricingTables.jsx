import React from 'react';
import Button from '../Atoms/Buttons.jsx';
import { trackEvent } from '../../lib/analytics.js';

const packages = [
  {
    name: 'Brons',
    subtitle: 'Entry-level pakket',
    price: '€495',
    duration: '4 uur showtijd',
    category: ['Private', 'Bedrijfsfeest'],
    features: [
      '4 uur DJ-set',
      'Basis licht- en geluidsset',
      'Persoonlijk intakegesprek',
      'Muziekvoorkeuren formulier',
    ],
    upsells: [
      { id: 'rookmachine', label: 'Rookmachine & sfeerlicht', price: '€75' },
      { id: 'ceremonie', label: 'Ceremonie audio set', price: '€195' },
    ],
    isFeatured: false,
    buttonText: 'Meer Info',
  },
  {
    name: 'Zilver',
    subtitle: 'Meest gekozen',
    price: '€795',
    duration: '5 uur showtijd',
    category: ['Bruiloft'],
    features: [
      '6 uur DJ-set',
      'Uitgebreide lichtshow',
      'DJ + Saxofonist optie',
      '100% dansgarantie',
      'Onbeperkt aantal gasten',
    ],
    upsells: [
      { id: 'sparkular', label: 'Sparkular show', price: '€245' },
      { id: 'sax', label: 'Live saxofonist', price: '€325' },
      { id: 'photobooth', label: '360° photobooth', price: '€425' },
    ],
    isFeatured: true,
    buttonText: 'Boek Nu',
  },
  {
    name: 'Goud',
    subtitle: 'Premium All-Inclusive',
    price: '€1.295',
    duration: '6 uur showtijd + productie',
    category: ['Bruiloft', 'Bedrijfsfeest'],
    features: [
      '8 uur DJ-set',
      'Luxe licht- en geluidsset',
      'DJ + Saxofonist (inbegrepen)',
      'Ceremonie & receptie muziek',
      'Professionele apparatuur',
    ],
    upsells: [
      { id: 'branding', label: 'Custom booth branding', price: '€395' },
      { id: 'host', label: 'Host & vocalist', price: '€295' },
      { id: 'afterfilm', label: 'Aftermovie & content', price: '€495' },
    ],
    isFeatured: false,
    buttonText: 'Vraag Offerte Aan',
  },
];

const PricingCard = ({ pkg }) => {
  const { name, subtitle, price, duration, category, features, upsells, isFeatured, buttonText } = pkg;

  const cardClasses = isFeatured
    ? 'bg-neutral-dark text-neutral-light shadow-2xl transform scale-105'
    : 'bg-neutral-light text-neutral-dark shadow-lg';

  const headerClasses = isFeatured
    ? 'text-secondary border-b border-secondary/50'
    : 'text-primary border-b border-neutral-gray-100';

  const buttonVariant = isFeatured ? 'secondary' : 'primary';

  const handleClick = () => {
    trackEvent('package_cta_click', {
      package_name: name,
      placement: 'design_system_pricing_tables',
    });
  };

  const handleHover = () => {
    trackEvent('package_view', {
      package_name: name,
      placement: 'design_system_pricing_tables',
    });
  };

  return (
    <div className={`relative flex flex-col p-spacing-xl rounded-lg transition duration-300 ${cardClasses}`}>
      {isFeatured && (
        <div className="absolute top-0 right-0 bg-secondary text-neutral-dark text-font-size-small font-bold px-spacing-md py-spacing-xs rounded-tr-lg rounded-bl-lg">
          Populair
        </div>
      )}
      <div className={`pb-spacing-md mb-spacing-md ${headerClasses}`}>
        <h3 className="text-font-size-h3 font-bold">{name}</h3>
        <p className="text-font-size-small opacity-80">{subtitle}</p>
      </div>
      <div className="flex flex-col mb-spacing-lg">
        <span className="text-font-size-h1 font-extrabold">{price}</span>
        {duration && <span className="text-font-size-small opacity-70">{duration}</span>}
        {category && (
          <span className="text-font-size-small opacity-80">
            Geschikt voor: {Array.isArray(category) ? category.join(', ') : category}
          </span>
        )}
      </div>
      <ul className="flex-grow space-y-spacing-sm mb-spacing-xl">
        {features.map((feature, index) => (
          <li key={index} className="flex items-start text-font-size-body">
            <svg
              className={`w-5 h-5 mr-spacing-sm ${isFeatured ? 'text-secondary' : 'text-primary'}`}
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M5 13l4 4L19 7" />
            </svg>
            {feature}
          </li>
        ))}
      </ul>
      {upsells && upsells.length > 0 && (
        <div className="mb-spacing-lg rounded-lg border border-primary/20 bg-primary/5 p-spacing-md">
          <p className="text-font-size-small font-semibold uppercase tracking-wide text-primary mb-spacing-xs">
            Populaire uitbreidingen
          </p>
          <ul className="space-y-spacing-xs text-font-size-small">
            {upsells.map((addon) => (
              <li key={addon.id} className="flex items-center justify-between">
                <span>{addon.label}</span>
                <span className="font-semibold">{addon.price}</span>
              </li>
            ))}
          </ul>
        </div>
      )}
      <Button
        variant={buttonVariant}
        size="lg"
        className="w-full"
        onClick={handleClick}
        onMouseEnter={handleHover}
      >
        {buttonText}
      </Button>
    </div>
  );
};

const PricingTables = () => {
  return (
    <section className="py-spacing-3xl bg-neutral-gray-100">
      <div className="container mx-auto px-spacing-md">
        <h2 className="text-font-size-h2 text-center text-neutral-dark mb-spacing-2xl font-extrabold">
          Onze Pakketten
        </h2>
        <p className="text-center text-font-size-body text-neutral-gray-600 max-w-2xl mx-auto mb-spacing-2xl">
          Combineer basispakketten met uitbreidingen zoals sparkulars, branding en live-musici. De data-layer events volgen elke
          klik zodat marketingteams kunnen optimaliseren.
        </p>
        <div className="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl items-start">
          {packages.map((pkg) => (
            <PricingCard key={pkg.name} pkg={pkg} />
          ))}
        </div>
      </div>
    </section>
  );
};

export default PricingTables;
