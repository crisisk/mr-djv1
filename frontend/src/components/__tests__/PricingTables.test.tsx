import { render } from '@testing-library/react';
import { PricingTables, type PricingFeatureSet } from '../PricingTables';

describe('PricingTables', () => {
  it('matches the snapshot for custom packages', () => {
    const packages: PricingFeatureSet[] = [
      {
        name: 'Essentials',
        subtitle: 'Voor verjaardagen en jubilea',
        price: '€550',
        features: ['4 uur DJ-set', 'Compacte lichtshow', 'Intakegesprek'],
        buttonText: 'Plan intake',
      },
      {
        name: 'Celebration',
        subtitle: 'Meest gekozen voor bruiloften',
        price: '€890',
        features: ['6 uur DJ-set', 'Live saxofonist', 'Premium geluid', 'Dansvloer verlichting'],
        isFeatured: true,
        buttonText: 'Boek nu',
      },
      {
        name: 'Grand Gala',
        subtitle: 'Voor grote events en bedrijfsfeesten',
        price: '€1.450',
        features: ['8 uur DJ-set', 'Licht- en ledshow', 'MC service', 'Aftermovie'],
        buttonText: 'Vraag offerte aan',
      },
    ];

    const { container } = render(<PricingTables packages={packages} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
