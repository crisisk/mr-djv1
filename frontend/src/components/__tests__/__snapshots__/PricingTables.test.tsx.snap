// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PricingTables matches the snapshot for custom packages 1`] = `
<section
  class="py-spacing-3xl bg-neutral-gray-100"
>
  <div
    class="container mx-auto px-spacing-md"
  >
    <h2
      class="heading-2 text-center text-neutral-dark mb-spacing-2xl"
    >
      Onze Pakketten
    </h2>
    <div
      class="grid grid-cols-1 md:grid-cols-3 gap-spacing-xl items-center"
    >
      <div
        class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-light text-neutral-dark shadow-lg"
      >
        <div
          class="pb-spacing-md mb-spacing-md text-primary border-b border-neutral-gray-100"
        >
          <h3
            class="heading-3"
          >
            Essentials
          </h3>
          <p
            class="body-sm opacity-80"
          >
            Voor verjaardagen en jubilea
          </p>
        </div>
        <div
          class="flex items-baseline mb-spacing-lg"
        >
          <span
            class="heading-1"
          >
            €550
          </span>
          <span
            class="body-md ml-spacing-xs"
          >
            / event
          </span>
        </div>
        <ul
          class="flex-grow space-y-spacing-sm mb-spacing-xl"
        >
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-primary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            4 uur DJ-set
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-primary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Compacte lichtshow
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-primary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Intakegesprek
          </li>
        </ul>
        <button
          class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-primary text-neutral-light hover:bg-primary/90 focus-visible:ring-primary/40 px-spacing-lg py-spacing-md body-lg w-full"
          type="button"
        >
          Plan intake
        </button>
      </div>
      <div
        class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-dark text-neutral-light shadow-2xl transform scale-105"
      >
        <div
          class="absolute top-0 right-0 bg-secondary text-neutral-dark body-sm text-strong px-spacing-md py-spacing-xs rounded-tr-lg rounded-bl-lg"
        >
          Populair
        </div>
        <div
          class="pb-spacing-md mb-spacing-md text-secondary border-b border-secondary/50"
        >
          <h3
            class="heading-3"
          >
            Celebration
          </h3>
          <p
            class="body-sm opacity-80"
          >
            Meest gekozen voor bruiloften
          </p>
        </div>
        <div
          class="flex items-baseline mb-spacing-lg"
        >
          <span
            class="heading-1"
          >
            €890
          </span>
          <span
            class="body-md ml-spacing-xs"
          >
            / event
          </span>
        </div>
        <ul
          class="flex-grow space-y-spacing-sm mb-spacing-xl"
        >
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-secondary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            6 uur DJ-set
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-secondary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Live saxofonist
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-secondary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Premium geluid
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-secondary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Dansvloer verlichting
          </li>
        </ul>
        <button
          class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-secondary text-neutral-dark hover:bg-secondary/90 focus-visible:ring-secondary/40 px-spacing-lg py-spacing-md body-lg w-full"
          type="button"
        >
          Boek nu
        </button>
      </div>
      <div
        class="relative flex flex-col p-spacing-xl rounded-lg transition duration-300 bg-neutral-light text-neutral-dark shadow-lg"
      >
        <div
          class="pb-spacing-md mb-spacing-md text-primary border-b border-neutral-gray-100"
        >
          <h3
            class="heading-3"
          >
            Grand Gala
          </h3>
          <p
            class="body-sm opacity-80"
          >
            Voor grote events en bedrijfsfeesten
          </p>
        </div>
        <div
          class="flex items-baseline mb-spacing-lg"
        >
          <span
            class="heading-1"
          >
            €1.450
          </span>
          <span
            class="body-md ml-spacing-xs"
          >
            / event
          </span>
        </div>
        <ul
          class="flex-grow space-y-spacing-sm mb-spacing-xl"
        >
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-primary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            8 uur DJ-set
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-primary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Licht- en ledshow
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-primary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            MC service
          </li>
          <li
            class="flex items-start body-md"
          >
            <svg
              class="w-5 h-5 mr-spacing-sm text-primary"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5 13l4 4L19 7"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            Aftermovie
          </li>
        </ul>
        <button
          class="inline-flex items-center justify-center rounded-md text-strong transition-colors duration-200 disabled:opacity-60 disabled:cursor-not-allowed focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 bg-primary text-neutral-light hover:bg-primary/90 focus-visible:ring-primary/40 px-spacing-lg py-spacing-md body-lg w-full"
          type="button"
        >
          Vraag offerte aan
        </button>
      </div>
    </div>
  </div>
</section>
`;
